db/migrate/20200713173228_add_priority_to_distributions.rb

```diff
@@ -0,0 +1,10 @@
+class AddPriorityToDistributions < ActiveRecord::Migration[5.2]
+  def up
+    add_column :distributions, :priority, :boolean, comment: "Whether or not this distribution was a priority appeals only push to judges via weekly job (not requested)"
```

Comment:
```suggestion
    add_column :distributions, :priority_pushed, :boolean, comment: "Whether or not this distribution was a result of a priority-appeals-only push to judges via a weekly job (not requested)"
```

Reply:
Re-reading this. Maybe this is more appropriate:
```suggestion
    add_column :distributions, :priority, :boolean, comment: "Whether or not this distribution is a priority-appeals-only push to judges via a weekly job (not manually requested)"
```